app/controllers/application_controller.rb

```diff
@@ -54,11 +54,9 @@ def verify_private_forem
     return if self.class.module_parent.to_s == "Admin"
     return if user_signed_in? || Settings::UserExperience.public
 
-    @page = Page.landing_page
-
     if api_action?
       authenticate!
-    elsif @page
+    elsif (@page = Page.landing_page)
```

Comment:
this was put here to avoid loading the landing page if the request is an API request...